### PR TITLE
Students have a toggleable help status that is displayed and controlled ...

### PR DESCRIPTION
**Pull request by pjfranzini**
...in header bar:

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -33,6 +33,7 @@
 
   get '/profile/edit', to: "users#edit_profile", as: "edit_profile"
   patch '/profile', to: 'users#update_profile', as: 'update_profile'
+  patch '/gethelp', to: 'students#update_status', as: 'update_status'
```
Comment (aomran) [inline config/routes.rb]: the path should probably be '/help_status/'. RESTful routes have a convention: the path describes the resource not the action. The action is described by the method (in this case PATCH).

**Comment by aomran** [comment]
Looks good.

**Comment by nachiketkumar** [comment]
So I just pulled these changes and now I'm getting this error on my app:

Migrations are pending; run 'bin/rake db:migrate RAILS_ENV=development' to resolve this issue.

In the past it's simply been run 'rake db:migrate' so I just want to make sure that I actually need to run this new command?

**Comment by pjfranzini** [comment]
For me after last pull I had to do 
bundle install
and then things worked. Not sure about the migrate/RAILS_ENV etc. Ahmed back in 10-15 minutes

**Comment by pjfranzini** [comment]
But I did add a column to a migration; and on my system at any rate I did simply
rake db:migrate

**Comment by aomran** [comment]
Same thing. Rake is actually in the bun folder, omitting the path is optional. And by default it knows what environment to use. So rake db:migrate will do the trick.

**Comment by aomran** [comment]
Bin... Stupid phone

**Comment by nachiketkumar** [comment]
Sounds good- just thought I'd check. works now :smile:

**Pull request merged**